test/e2e-common.sh

```diff
@@ -372,3 +374,14 @@ function wait_for_file() {
   done
   return 0
 }
+
+function install_cert_manager(){
+  url="$(curl -s https://api.github.com/repos/cert-manager/cert-manager/releases/latest | jq '.assets[] | select(.name|match("cert-manager.yaml$")) | .browser_download_url')"
+
+  kubectl apply \
+    -f "${url}" || return 1
+  
+  scale_controlplane cert-manager cert-manager-cainjector cert-manager-webhook
```

Comment:
We don't need to scale cert-manager, we scale Knative pods to test for Knative HA but cert-manager testing is already done/left to the cert manager project.
```suggestion
```